src/reporter/labels.js

```diff
@@ -3,91 +3,91 @@ export const DEPRECATED_MANAGED_BY_OTA_MARKER = '[managed by OTA]';
 export const MANAGED_BY_OTA_MARKER = '- Auto-managed by OTA engine';
 
 export const LABELS = {
+  DNS_LOOKUP_FAILURE: {
+    name: 'domain lookup failed',
+    color: 'FFFFFF',
+    description: 'Fetching failed because the domain temporarily failed to resolve on DNS',
```

Comment:
I don't understand the difference with `DNS_RESOLUTION_FAILURE`. If neither do you, could we maybe use the same description for both?

Reply:
`DNS_LOOKUP_FAILURE` is related to `EAI_AGAIN` error which is typically due to a network issue or a timeout reaching the DNS server. Whereas `DNS_RESOLUTION_FAILURE` si related to `ENOTFOUND` error which means that the DNS server responded, but the domain does not exist.

See https://betterstack.com/community/guides/scaling-nodejs/nodejs-errors